frontend/src/js/external-forms/form-concept-group/FormConceptGroup.tsx

```diff
@@ -199,6 +203,58 @@ const FormConceptGroup = (props: Props) => {
             ? t("externalForms.common.concept.copying")
             : props.attributeDropzoneText
         }
+        dropBetween={(i: number) => {
+          return (item: DragItemConceptTreeNode) => {
+            if (props.isValidConcept && !props.isValidConcept(item))
+              return null;
+
+            if (isMovedObject(item)) {
+              let insertIndex =
+                i > item.dragContext.movedFromAndIdx &&
+                item.dragContext.movedFromOrIdx === 0
+                  ? i - 1
+                  : i;
+              if (item.dragContext.movedFromFieldName === props.fieldName) {
+                const updatedValue =
+                  props.value[item.dragContext.movedFromAndIdx].concepts
+                    .length === 1
+                    ? removeValue(props.value, item.dragContext.movedFromAndIdx)
+                    : removeConcept(
+                        props.value,
+                        item.dragContext.movedFromAndIdx,
+                        item.dragContext.movedFromOrIdx,
+                      );
+                return props.onChange(
+                  addConcept(
+                    insertValue(updatedValue, insertIndex, newValue),
+                    insertIndex,
+                    copyConcept(item),
+                  ),
+                );
+              } else {
+                if (exists(item.dragContext.deleteFromOtherField)) {
+                  item.dragContext.deleteFromOtherField();
+                }
```

Comment:
Could you simplify this code (and the other / similar part) to make it more readable? Maybe by destructuring the variables from dragContext?